src/textual/widgets/_markdown.py

```diff
@@ -589,7 +589,7 @@ def __init__(self, href: str) -> None:
 
     async def on_mount(self) -> None:
```

Comment:
Make method non-async?

```suggestion
    def on_mount(self) -> None:
```